xcode.mk

```diff
@@ -7,10 +7,19 @@ CMAKE_BOOST_OPTIONS = -DBoost_NO_BOOST_CMAKE=TRUE \
 	-DBOOST_ROOT="$(BOOST_ROOT)"
 endif
 
+# https://cmake.org/cmake/help/latest/envvar/MACOSX_DEPLOYMENT_TARGET.html
+export MACOSX_DEPLOYMENT_TARGET = 10.9
```

Comment:
Actually I were not sure if I should specify this option for librime. It works anyway without it.
Are there any information that tells the impact of not having this option?

Reply:
Without this environment variable set, there will be a lot warnings like:

```
Warning: librime.1.dylib was built for newer OSX version (10.11) than being linked (10.9)
```

the thirdparty libraries too.

#402